Assets/FullInspector2/Modules/InspectorCurve/Editor/InspectorCurveAttributeEditor.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using UnityEditor;
+using UnityEngine;
+
+namespace FullInspector.Modules {
+    [CustomAttributePropertyEditor(typeof(InspectorCurveAttribute), ReplaceOthers = true)]
+    public class InspectorCurveAttributeEditor<TElement> : AttributePropertyEditor<TElement, InspectorCurveAttribute> {
```

Comment:
Since `AnimationCurve` is a sealed class, you don't need to make this editor generic. So

`public class InspectorCurveAttributeEditor : AttributePropertyEditor<AnimationCurve, InspectorCurveAttribute> {`